### PR TITLE
chatty: Add version 0.13.1

### LINKED ISSUE #5114: [Request] Add Chatty
https://github.com/chatty/chatty

Thanks!

### PR DESCRIPTION
**Pull request by earnestma**
- Closes #5114

### CODE REVIEW

File: bucket/chatty.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            "hash": "c5c39de62db3d853668a16f8044481bde5b3b3344b90d02ac18e4226d5425a93"
+        }
+    },
+    "bin": "Chatty.exe",
+    "shortcuts": [
+        [
+        "Chatty.exe",
+        "Chatty"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/chatty.json:20]:
```suggestion
    ],
    "persist": [
        "app\\portable_settings",
        "app\\Chatty.cfg"
    ],
```

File: bucket/chatty.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "license": {
+        "identifier": "GPL-3.0-or-later"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/chatty/chatty/releases/download/v0.13/Chatty_0.13_win_standalone.zip",
+            "hash": "c5c39de62db3d853668a16f8044481bde5b3b3344b90d02ac18e4226d5425a93"
+        }
+    },
+    "bin": "Chatty.exe",
```
Comment (Ash258) [inline bucket/chatty.json:14]:
```suggestion
    "pre_install": [
        "if (!(Test-Path \"$persist_dir\\app\\Chatty.cfg\")) {",
        "    ensure \"$dir\\app\\portable_settings\" | Out-Null",
        "    $cfg = \"$dir\\app\\Chatty.cfg\"",
        "    (Get-Content $cfg) -replace '(\\[ArgOptions\\])', \"`$1`r`n-portable\" | Set-Content $cfg -Encoding Ascii -Force",
        "    Set-Content \"$dir\\app\\portable_settings\\settings\" '{ \"checkNewVersion\": false }' -Encoding Ascii",
        "}"
    ],
    "bin": "Chatty.exe",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by Ash258** [comment]
Your contributions will not be accepted if you will not follow github flow.

Next time you will commit changes not from UI using suggestions and without giving credits to review your all your PRs will be closed.

File: bucket/chatty.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "0.13",
+    "description": "Twitch Chat Client written in Java",
+    "homepage": "https://chatty.github.io/",
```
Comment (Ash258) [inline bucket/chatty.json:4]:
```suggestion
    "description": "Twitch Chat Client",
    "homepage": "https://chatty.github.io",
```

File: bucket/chatty.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        "app\\portable_settings",
+        "app\\Chatty.cfg"
+    ],
+    "checkver": {
+        "github": "https://github.com/chatty/chatty"
+    },
+    "autoupdate": {
+        "url": "https://github.com/chatty/chatty/releases/download/v$version/Chatty_$version_win_standalone.zip",
+        "hash": {
+            "url": "https://tduva.com/chatty/checksums/v$version/Chatty_$version_win_standalone.zip.sha256"
```
Comment (Ash258) [inline bucket/chatty.json:33]: Slightly better

```suggestion
            "url": "https://tduva.com/chatty/checksums/v$version/$basename.sha256"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by earnestma** [comment]
Very sorry about that. How should I allow your write access to PRs?

**Comment by earnestma** [comment]
nvm, it looks like [that option is only available for user forks](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) (I usually fork to an org). Next time I'll use a personal fork instead

**Review by Ash258 — APPROVED** [review]

**Pull request merged**